aladdin/lib/arg_tools.py

```diff
@@ -55,3 +56,37 @@ def add_command(parser, cmd=bash_cmd, help_msg=bash_help["help"]):
             sub_parser.set_defaults(func=lambda args: bash_wrapper())
         commands.append((bash_cmd, add_command))
     return commands
+
+
+HELM_OPTION_PARSER = argparse.ArgumentParser(add_help=False)
+HELM_OPTION_PARSER.add_argument(
+    "--chart", help="which chart in the project to use, defaults to the project name"
+)
```

Comment:
too bad we have to lose the more specified multi-chart functionality for the `start` command. The docs need to be updated to reflect this change.

Reply:
I've actually been making use of it lately. Would it be difficult to keep?

Reply:
not at all, turned out to be pretty simple actually, done!